connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -44,6 +44,40 @@ import org.apache.spark.sql.types._
 
 class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper with PrivateMethodTester {
 
+  test(s"throw SparkException with large cause exception") {
+    withSQLConf("spark.sql.spark.enrichError.enabled" -> "true") {
```

Comment:
Can we use `Connect.SPARK_ENRICH_ERROR_ENABLED.key` to get the key

Reply:
No if I understand correctly since Connect config is in the server module and client module does not depend on the server module

Reply:
Connect is in connect/common... so you can use it.

Reply:
Do you mean [this connect](https://github.com/apache/spark/blob/171251518b469824589c498c5f202cc55dacb128/connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala)? It is in `connect/server`?